src/assets/styles/tools/mixins.scss

```diff
@@ -59,33 +28,45 @@
   }
 }
 
-// Responsive Show/Hide Helper
-@mixin responsive-display-helper($breakpoint: '') {
-  .#{$breakpoint}show {
-    display: block !important;
-  }
+/**
+ * Outputs list of transition properties
+ *
+ * Accepts:
+ * - $transitions: {String} List of transition properties to set
+ *
+ * Usage:
+ * .selector {
+ *   @include transition(width, height 0.5s ease-in);
+ * }
+ */
+@mixin transition($transitions...) {
```

Comment:
Overall, what does it do? Is there a reason we can't use the native way instead of a mixin?

Reply:
I'm okay with it! 👍